Drivers/Src/Switches.c

```diff
@@ -1,7 +1,12 @@
 #include "Switches.h"
+#include "stm32f4xx.h"
```

Comment:
Noted. Keeping it to the stm32 specific stuff for right now because we know it works. Once Lights and Switches are merged we can go through and replace with BSP stuff. Also, the BSP_GPIO_Write_Pin function in BSP_GPIO.c is not correctly implemented, so I'm going to fix that.